### PR TITLE
Tweak recomends and empty state logic

### PR DESCRIPTION
**Pull request by phamann**
Fixes a few issues with placement of recommends a/b slice and empty state card on the time view:

- Ensure recommends slice appears if less than 4 items of content
- Ensure empty state appears if less than 4 items of content
- Ensure empty state is 3 items apart from recommends when > 4 bits of content

google chrome

### CODE REVIEW

**Comment by matthew-andrews** [comment]
podcasts image looking good in that screenshot :see_no_evil:

**Comment by laurieboyes** [comment]
:+1:

File: server/helpers/index.js [hunk 1]
```diff
@@ -2,9 +2,11 @@
 import reactRenderToString from './react-render-to-string';
 import colspan from './colspan';
 import ifThird from './if-third';
+import ifIndex from './if-index';
 
 export default {
     reactRenderToString,
     colspan,
-    ifThird
+    ifThird,
```
Comment (keirog) [inline server/helpers/index.js]: Still using ifThird?

**Comment by keirog** [comment]
Other than ifThird, :+1:

**Pull request merged**